### ISSUE TITLE
[3.9] Multiple jumpstart scripts are created

### ISSUE THREAD

**Issue by Informatic** [RootMyTV/RootMyTV.github.io#27]
`HE_DTV_W17R_AFAAABAA`, `06.00.10` Software Version.

Summary of a saga from: https://github.com/webosbrew/youtube-webos/issues/30

Somehow "legacy" (v1) jumpstart script write errored out, while still writing a file properly, leaving a system with multiple jumpstarts, causing Failsafe Mode popup to show up on every boot.

**Comment by stek29** [comment]
at least for me there’s only one jumpstart script, but it gets executed twice:

```

root@living-room-lgtv:~# ls -l /media/cryptofs/apps/usr/palm/services/com.palmdts.devmode.service
/start-devmode.sh
ls: /media/cryptofs/apps/usr/palm/services/com.palmdts.devmode.service/start-devmode.sh: No such file or directory
root@living-room-lgtv:~# head -n3 /mnt/lg/cmn_data/wam/extra_conf.sh

pstree -aslp $$ >> /tmp/extraup.txt

root@living-room-lgtv:~# cat /tmp/extraup.txt 
init,1
  `-sh,2941 -e /proc/self/fd/9
      `-pstree,2977 -aslp 2941
init,1
  `-sh,3698 -e /proc/self/fd/9
      `-pstree,3733 -aslp 3698
root@living-room-lgtv:~# head -n3 /var/lib/webosbrew/startup.sh
#!/bin/bash

pstree -aslp $$ >> /tmp/brewup.txt
root@living-room-lgtv:~# cat /tmp/brewup.txt 
init,1
  `-WebAppMgr,2941 --application-cache-domain-limit=10485760 --application-cache-size=52428800 --browser-subprocess-path=/usr/bin/WebAppMgr --disable-direct-npapi-requests --disable-extensions --disable-gpu-shader-disk-cache --disable-low-res-tiling --disable-new-video-renderer --disk-cache-size=52428800 --enable-aggressive-domstorage-flushing --enable-aggressive-release-policy --enable-devtools-experiments --enable-gpu-rasterization --disable-gpu-rasterization-for-first-frame --enable-impl-side-painting --enable-key-event-throttling --enable-pinch-virtual-viewport --enable-threaded-compositing --enable-watchdog --ignore-gpu-blacklist --ignore-netif=p2p --in-process-gpu --log-level-to-pmlog=2 --max-unused-resource-memory-usage-percentage=0 --network-stable-timeout=10 --noerrdialogs --num-raster-threads=2 --ozone-platform=wayland --remote-debugging-port=9998 --resource-buffer-max-allocation-size=262144 --resource-buffer-size=1048576 --touch-events=disabled --ui-disable-opaque-shader-program --ui-use-high-precision-shader-program --user-data-dir=/var/lib/wam --v8-use-lightweight-gc-on-idle --enable-css-map-to-mesh-filter --webos-wam \\ --disable-v8-crankshaft --no-zygote \\ --skia-font-cache-size-mb=1 --skia-image-cache-size-mb=40 --skia-background-font-cache-size-kb=0 --gpu-program-cache-size-kb=2048 --watchdog-render-timeout=200 --enable-local-resource-code-cache --disallow-code-cache-from-file-uris-with-query-string --user-agent-spoof=/usr/palm/applications/com.webos.app.browser --js-flags=--ignition --noexpose_wasm
      `-exe,3054 /var/lib/webosbrew/startup.sh
          `-pstree,3063 -aslp 3054
init,1
  `-WebAppMgr,3698 --application-cache-domain-limit=10485760 --application-cache-size=52428800 --browser-subprocess-path=/usr/bin/WebAppMgr --disable-direct-npapi-requests --disable-extensions --disable-gpu-shader-disk-cache --disable-low-res-tiling --disable-new-video-renderer --disk-cache-size=52428800 --enable-aggressive-domstorage-flushing --enable-aggressive-release-policy --enable-devtools-experiments --enable-gpu-rasterization --disable-gpu-rasterization-for-first-frame --enable-impl-side-painting --enable-key-event-throttling --enable-pinch-virtual-viewport --enable-threaded-compositing --enable-watchdog --ignore-gpu-blacklist --ignore-netif=p2p --in-process-gpu --log-level-to-pmlog=2 --max-unused-resource-memory-usage-percentage=0 --network-stable-timeout=10 --noerrdialogs --num-raster-threads=2 --ozone-platform=wayland --remote-debugging-port=9998 --resource-buffer-max-allocation-size=262144 --resource-buffer-size=1048576 --touch-events=disabled --ui-disable-opaque-shader-program --ui-use-high-precision-shader-program --user-data-dir=/var/lib/wam --v8-use-lightweight-gc-on-idle --enable-css-map-to-mesh-filter --webos-wam \\ --disable-v8-crankshaft --no-zygote \\ --skia-font-cache-size-mb=1 --skia-image-cache-size-mb=40 --skia-background-font-cache-size-kb=0 --gpu-program-cache-size-kb=2048 --watchdog-render-timeout=200 --enable-local-resource-code-cache --disallow-code-cache-from-file-uris-with-query-string --user-agent-spoof=/usr/palm/applications/com.webos.app.browser --js-flags=--ignition --noexpose_wasm
      `-exe,3764 /var/lib/webosbrew/startup.sh
          `-pstree,3771 -aslp 3764
```

**Comment by Informatic** [comment]
A fix/workaround for this has been released in Homebrew Channel v0.5.0 - this will be set as a default installed version ~this weekend, after we confirm no issues with that version..
